### ISSUE TITLE
Подсвечивать изменения в новых комментариях

### ISSUE THREAD

**Issue by Ori-Lama** [lunavod/ponydays#116]
Только сейчас понял откуда лишние желтые комментарии берутся.
Раньше он еще и прокликивался как новый, но сейчас норм. Только подсвечивается.

**Comment by lunavod** [comment]
Лооол. То что он сейчас не прокликивается - наоборот баг. Должен был по моей задумке прокликиваться.

А что, думаешь, это лишнее? Ибо ну вот если человек обновил коммент, то ты это можешь и не заметить. А там что-то важное измениться могло.

**Comment by Ori-Lama** [comment]
Ну да, важное может быть, я как-то не подумал. Но обычно изменения настолько маленькие, что и не заметны. Запятая там или буква какая-то меняется. А щелкалка заставляет к этой измененной букве перематываться лишний раз. Хотя в принципе идея у тебя правильная - изменение ведь может быть больше чем одна буква и важным.

Да, наверное так оно и лучше. Пусть они прокликиваются и подсвечиваются как новые.

**Comment by Ori-Lama** [comment]
О, а может быть как-то выделять то, что было отредактировано? Новые дописанные слова, например. Чтобы при одном взгляде было понятно что изменилось. А то ведь обычно текст остается почти тем же, и можно на автомате проглядеть и не заметить.

**Comment by lunavod** [comment]
Хм, а это вот как раз интересно.

**Comment by Ori-Lama** [comment]
Во, а когда прокликнешь сообщение как прочитанное, то это выделение пусть убирается, чтобы потом не мельтешило. Ну, если такое возможно вообще.

**Comment by lunavod** [comment]
Только вопрос, как это сделать. Ибо HTML. Т.е. казалось бы, все просто - берем да и оборачиваем все изменения в спан с классом. Но а если, например, изменение было в теге? Т.е. например при написании коммента сслучайно затер одну букву в теге спойлера, потом отредактировал. Если попытаться обернуть изменение в тег, то это сломает верстку. D:

**Comment by Ori-Lama** [comment]
Это беда, да. Ну, значит что-то типа IF изменения не в теге: сделать подсветку текста.

**Comment by lunavod** [comment]
Ну вот проблема в том, как это распарсить...

**Comment by Ori-Lama** [comment]
Ну IF слева < и справа >: значит мы в теге )

**Comment by lunavod** [comment]
Ага... `Хм, значит, если x<y, то y>x?`
И все сломалось. :D

**Comment by Ori-Lama** [comment]
То есть эти < > не хранятся как `&lt; &gt;`? Я-то думал, что хранятся.

**Comment by Ori-Lama** [comment]
Ну да, хранится исходный код коммента, а в нем не обязательны lt gt.

**Comment by lunavod** [comment]
А, лол. Нифига, то что я там написал распарсится криво в принципе.

**Comment by lunavod** [comment]
Да, туплю. Такое сработает только в code, а в нем автоматически экранируется...

**Comment by lunavod** [comment]
https://github.com/kpdecker/jsdiff
Думаю, стоит использовать вот этот модуль.
